api/src/main/java/org/pmiops/workbench/api/RuntimeController.java

```diff
@@ -194,7 +193,7 @@ public ResponseEntity<EmptyResponse> createRuntime(String workspaceNamespace, Ru
               LEONARDO_LABEL_IS_RUNTIME_TRUE));
     }
     long configCount =
-        Stream.of(runtime.getGceConfig(), runtime.getDataprocConfig(), runtime.getGceWithPdConfig())
```

Comment:
while I believe GCE is OK to remove, I am not 100% confident... let's take a closer look.

Reply:
I tried out in UI with @yonghaoy , we didn't find a way to create non PD enabled VM (other than dataproc)